### DISCUSSION TITLE
Storage Provider Working Group

### DISCUSSION THREAD

**Discussion by Byeager1** [filecoin-project/community#226] [Mining]
Following up per the last NA Storage Provider Working Group ("SPWG") meeting.  We discussed the cadence of how the SPWGs and broader community could collaborate to determine tooling and development priorities to improve the storage provider experience and vote on issuing bounties for development teams to help bring these items into fruition.

We were thinking of collaborating with the overall community to determine the top 5 most common tools we would like to be built and current issues we would like to see fixed, then voting (within the SPWG) on bounty structure (amount of FIL, timelines, etc) to see if we can incentivize development teams to build the tools for us.  

Please let us know you thoughts on this and how we can work together to get this started.

**Comment by flyworker** [comment]
Here are something comes to my mind what we need for development:
* node function splitting - we know we have it but maybe  we need more detailed features
* Scheduling P1 P2 C2 binding  to different machines
* Alert system for miss winpost/slow in system

We know there are dev grants, but we also need a developer dedicated to the optimization

**Reply by kemihu11** [reply]
+1 to @flyworker
